helmfile/upstream/dex/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
```

Comment:
This file looks like a mashup between 
https://github.com/helm/charts/blob/master/stable/dex/Chart.yaml and
https://github.com/dexidp/helm-charts/blob/dex-0.3.3/charts/dex/Chart.yaml